test/evergreen/find_cmake.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+set -o errexit  # Exit the script with error if any of the commands fail
+
+# Adapted 'find_cmake' from mongo-c-driver evergreen infrastructure:
+#   https://github.com/mongodb/mongo-c-driver/blob/master/.evergreen/find-cmake.sh
+find_cmake ()
+{
+  if [ ! -z "$CMAKE" ]; then
+    return 0
+  elif [ -f "/Applications/Cmake.app/Contents/bin/cmake" ]; then
+    CMAKE="/Applications/Cmake.app/Contents/bin/cmake"
+    CTEST="/Applications/Cmake.app/Contents/bin/ctest"
+  elif [ -f "/opt/cmake/bin/cmake" ]; then
+    CMAKE="/opt/cmake/bin/cmake"
+    CTEST="/opt/cmake/bin/ctest"
+  elif command -v cmake 2>/dev/null; then
```

Comment:
Shouldn't you prefer the one in the `PATH`? I'd have expected this check to be the first one.

Reply:
Yep, that's a good point. `PATH` should be preferred. I've move the check above the others.